src/Types/Util.hs

```diff
@@ -201,6 +201,9 @@ isAdmin = (`hasRole` Admin)
 isStudent :: User -> Bool
 isStudent u = isJust $ find (has _Student) (u ^.. userRoles)
```

Comment:
`warn = isJust . find ==> any`

(not related to this PR, but we might as well "fix" it instead of keeping track of it.)